diff --git a/Pods/AFOAuth1Client/AFOAuth1Client/AFOAuth1Client.m b/Pods/AFOAuth1Client/AFOAuth1Client/AFOAuth1Client.m
index 8e4fdef..25a4723 100644
--- a/Pods/AFOAuth1Client/AFOAuth1Client/AFOAuth1Client.m
+++ b/Pods/AFOAuth1Client/AFOAuth1Client/AFOAuth1Client.m
@@ -398,7 +398,7 @@ static NSDictionary * AFKeychainQueryDictionaryWithIdentifier(NSString *identifi
 
     // Only use parameters in the HTTP POST request body (with a content-type of `application/x-www-form-urlencoded`).
     // See RFC 5849, Section 3.4.1.3.1 http://tools.ietf.org/html/rfc5849#section-3.4
-    NSDictionary *authorizationParameters = ([[request valueForHTTPHeaderField:@"Content-Type"] isEqualToString:@"application/x-www-form-urlencoded"] ? parameters : nil);
+    NSDictionary *authorizationParameters = ([[request valueForHTTPHeaderField:@"Content-Type"] hasPrefix:@"application/x-www-form-urlencoded"] ? parameters : nil);
     [request setValue:[self authorizationHeaderForMethod:method path:path parameters:authorizationParameters] forHTTPHeaderField:@"Authorization"];
     [request setHTTPShouldHandleCookies:NO];
     
